src/cascadia/TerminalSettingsModel/ApplicationState.cpp

```diff
@@ -156,87 +144,249 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     void ApplicationState::Reset() noexcept

```

Comment:
we may want to document why this doesn't reset Elevated

Reply:
that's unintentional